gradle.properties

```diff
@@ -1,4 +1,5 @@
-#Mon, 18 Jun 2018 18:59:18 +0200
 programmingVersion=8.3.0-SNAPSHOT
 schedulingVersion=8.3.0-SNAPSHOT
 schedulingSerialver=830L
+
+version.cas.client=3.5.0
```

Comment:
the org.jasig.cas.client is a standard library dependency, I don't understand why the version must be set in the gradle.properties files instead of directly in the build.gradle file